README.md

```diff
@@ -1,2 +1,38 @@
-# dagster-utils
-Utilities and reusable patterns for using Dagster for Broad pipeline orchestration
+# Dagster Utils
+
+A collection of common utilities used by the Monster team to develop Dagster pipelines. Each subsection has its own readme explaining its contents.
+
+## How to use this library
+
+This library is hosted on the [Broad Artifactory](https://broadinstitute.jfrog.io/). To configure Poetry to check that repository for packages, follow the instructions on private repositories [in their docs](https://python-poetry.org/docs/repositories/#install-dependencies-from-a-private-repository).
```

Comment:
We shouldn't need to auth to pull down the package, our stuff is all publicly accessible. For example, ingest-utils available anonymously (i.e., curl --output foo.jar https://broadinstitute.jfrog.io/broadinstitute/libs-release/org/broadinstitute/monster/scio-utils_2.12/1.4.0/scio-utils_2.12-1.4.0.jar)